packages/php-platform/.github/workflows/build-tagged-version.yaml

```diff
@@ -0,0 +1,28 @@
+on:
+    push:
+        tags:
+            - '**'
+    workflow_dispatch:
+name: Build PHP Platform Docker image for released version
+jobs:
+    rebuild-php-fpm-images:
+        name: Build tag
+        runs-on: ubuntu-22.04
+        permissions:
+            contents: read
+        steps:
+            -   name: GIT checkout tag - ${{ github.ref }}
+                uses: actions/checkout@v3
+            -   name: Login to DockerHub
+                uses: docker/login-action@v2
+                with:
+                    username: ${{ secrets.DOCKERHUB_USERNAME }}
+                    password: ${{ secrets.DOCKERHUB_PASSWORD }}
+            -   name: Set up QEMU
+                uses: docker/setup-qemu-action@v2
+            -   name: Docker Setup Buildx
```

Comment:
Why is this necessary?

Reply:
buildx is necessary to be able to build image for multiple platforms